src/hooks/useSparkleFirestoreConnect.ts

```diff
@@ -21,10 +22,17 @@ export type Defined<T> = T & Exclude<T, undefined>;
  * @see useSparkleFirestoreConnect
  * @see ReduxFirestoreQuerySetting
  */
-export interface SparkleRFQConfig extends ReduxFirestoreQuerySetting {
+
+export interface SparkleRFQConfigObject extends ReduxFirestoreQuerySetting {
   storeAs?: ValidFirestoreKeys;
 }
 
+export type SparkleRFQConfig =
+  | SparkleRFQConfigObject
+  | SparkleRFQConfigObject[]
+  | ValidFirestoreKeys // @debt Eridicate this as a way to connectData
+  | undefined;
```

Comment:
I wouldn't include this here, it needlessly hides away important information, and there are better choices for how to handle an explicit undefined where we want it.

```suggestion
```